File: DSA/VenikaPanwar_590013755/Day_20_Question_3.java
import java.util.Scanner;

public class KthLargestElement {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of elements: ");
        int n = sc.nextInt();
        int[] arr = new int[n];

        System.out.println("Enter " + n + " integers:");
        for (int i = 0; i < n; i++) arr[i] = sc.nextInt();

        System.out.print("Enter K (to find Kth largest): ");
        int k = sc.nextInt();

        // Selection sort in descending order
        for (int i = 0; i < k; i++) {
            int maxIndex = i;
            for (int j = i + 1; j < n; j++) {
                if (arr[j] > arr[maxIndex]) {
                    maxIndex = j;
                }
            }
            // Swap
            int temp = arr[i];
            arr[i] = arr[maxIndex];
            arr[maxIndex] = temp;
        }

        System.out.println("Kth largest element is: " + arr[k - 1]);
    }
}

File: DSA/Vritika_590012956/Day_20_Question_3.py
def find_kth_largest():
    arr = list(map(int, input("Enter the array elements separated by space: ").split()))
    k = int(input("Enter the value of K: "))

    # Sort in descending order
    arr.sort(reverse=True)

    print(f"The {k}th largest element is: {arr[k - 1]}")
find_kth_largest()
